next-tavla/app/(admin)/edit/[id]/components/TileCard/index.tsx

```diff
@@ -247,48 +261,77 @@ function TileCard({
                         onSubmit={reset}
                         onInput={() => setChanged(true)}
                     >
-                        <Heading4 margin="none">Gåavstand</Heading4>
+                        <Heading4 margin="bottom">Gåavstand</Heading4>
                         <SubParagraph>
                             Vis gåavstand fra lokasjonen til Tavla til
-                            stoppestedet
+                            stoppestedet.
                         </SubParagraph>
-                        <div className="flex flex-col">
-                            {!address?.name && (
-                                <Label className="!text-error">
-                                    {demoBoard
-                                        ? 'Logg inn for å få tilgang til funksjonaliteten'
-                                        : 'Du må legge til en lokasjon for å kunne skru på gåavstand'}
-                                </Label>
+                        {!address && (
+                            <Label className="!text-error">
+                                {demoBoard
+                                    ? 'Logg inn for å få tilgang til funksjonaliteten.'
+                                    : 'Du må legge til en lokasjon for å kunne skru på gåavstand.'}
+                            </Label>
+                        )}
+                        <Switch
+                            name="showDistance"
+                            disabled={!address}
+                            defaultChecked={
+                                tile.walkingDistance?.visible ?? false
+                            }
+                        >
+                            Vis gåavstand
+                        </Switch>
+
+                        <Heading4>Forskyv avgangstid</Heading4>
+                        <div className="flex flex-col gap-2">
+                            <SubParagraph>
+                                Vis kun avganger som går om mer enn et valgt
+                                antall minutter.
+                            </SubParagraph>
+                            <TextField
+                                label="Antall minutter"
+                                name="offset"
+                                id="offset"
+                                type="number"
+                                min={0}
+                                className="!w-2/5"
+                                value={offset}
+                                onChange={(e) => {
+                                    isNaN(e.target.valueAsNumber)
+                                        ? setOffset('')
+                                        : setOffset(e.target.valueAsNumber)
+                                }}
```

Comment:
Lurer på om vi kunne forenklet dette

```tsx
setOffset(e.target.valueAsNumber || 0)
```

Reply:
Eventuelt hvis vi ønsker at offset skal være undefined istedenfor:

```tsx
setOffset(e.target.valueAsNumber || undefined)
```

Reply:
det vil ikke funke å sette en `undefined` verdi - det er ikke tillatt å sende inn `undefined` eller `null` til controlled components: https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable

Reply:
grunnen til at jeg ikke har satt 0 er fordi det da vil se ut som man har satt "0" i input-feltet, når man egentlig ikke har gitt det en verdi. Det vil da se slik ut: 
![image](https://github.com/user-attachments/assets/4fbebd01-5248-4120-a29f-1208f5ada265)

i stedet for slik: 
![image](https://github.com/user-attachments/assets/6ee45a20-adbb-492c-ab67-0edf60cd06e3)